### PR TITLE
Revert "Merge /careers and /team into /who-we-are"

### PR DESCRIPTION
**Pull request by ElenaFdR**
I'm going to revert #99 and fix it. I've updated #99 with the errors I've spotted so far.

-----
[View rendered careers/index.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/careers/index.md)
[View rendered team/alba-roza.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/alba-roza.md)
[View rendered team/ben-cerveny.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/ben-cerveny.md)
[View rendered team/boris-van-hoytema.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/boris-van-hoytema.md)
[View rendered team/claus-mullie.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/claus-mullie.md)
[View rendered team/elena-findley-de-regt.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/elena-findley-de-regt.md)
[View rendered team/eric-herman.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/eric-herman.md)
[View rendered team/felix-faassen.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/felix-faassen.md)
[View rendered team/index.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/index.md)
[View rendered team/jan-ainali.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/jan-ainali.md)
[View rendered team/laura-scheske.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/team/laura-scheske.md)
[View rendered who-we-are/index.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/who-we-are/index.md)
[View rendered who-we-are/team/deborah-meibergen.md](https://github.com/publiccodenet/publiccode.net/blob/revert-99-Who-we-are/who-we-are/team/deborah-meibergen.md)

### CODE REVIEW

File: script/test.sh [hunk 1]
```diff
@@ -7,6 +7,5 @@ bundle exec jekyll build
 # Check for broken links and missing alt tags, ignore edit links to GitHub as they might not exist yet
 
 bundle exec htmlproofer \
-    --url-ignore "/github.com/(.*)/edit/" \
-    --url-ignore "/#/" \
```
Comment (ElenaFdR) [inline script/test.sh:11]: @ericherman We added this in #130 to unblock the new functionality added in #110, which is needed for #111 to render. @angelplasma says it can probably be made more precise - do think that's necessary?

File: script/test.sh [hunk 2]
```diff
@@ -7,6 +7,5 @@ bundle exec jekyll build
...
-    ./_site
\ No newline at end of file
+    --url-ignore "/github.com\/(.*)\/edit/,/twitter.com/" \
```
Comment (ElenaFdR) [inline script/test.sh:10]: Adding Twitter here will supersede #129.

File: script/test.sh (hunk 1 above)
Comment (ericherman) [reply script/test.sh:11]: I removed it because it was not required to make any current link pass, and matching any link with an anchor tag or '#' in it is far too broad. Looking at other branches I saw that twitter links were failing, thus added that in to be ahead of that approaching need.

Indeed, if we need to introduce something to make links pass in another branch, let's see what sorts of links are failing and match something more specific than '#'.

**Comment by angelplasma** [comment]
Hi @ericherman - thank you for jumping in on this!

**TL;DR:**
Would something like "#_" be more palatable as an exception?

**Background:**
On the upgraded Codebase description pages we're adding to the public site, simple CSS-based lightbox functionality is used to display a product screenshot gallery. The lightbox code makes use of content-less anchor tags to enable some UI to function.

**Further explanation:**
My (limited) understanding of Travis is that the `"/#/"` above would match an href containing exactly "#", but not any URL string simply containing a hash. Since `href="#"` is considered valid HTML in this context, and Travis seems to be tripping on it, the above seemed like a valid exception.

**Caveats/alternatives:**
1. If you would find exactly matching "#" acceptable, but the syntax in `"/#/"` is simply incorrect, perhaps this is a trivial fix.
2. If you'd prefer the exception was for a more idiosyncratic string, would something unusual like "#_" be more palatable?
3. If this direction is simply unworkable, we can potentially explore alternative solutions.

**Comment by ericherman** [comment]
@angelplasma My sense is that "#_" would be better yet that also seems too broad as we now have the hidden trap of not checking a link like "example.com/internal_functions#_foo". I believe to match exactly "#" the regular expression would require anchors on either side, e.g.: "/^#$/" as described in https://ruby-doc.org/core-2.7.1/Regexp.html

Is there a branch pushed which has the types of links which are failing which we can test against?

**Comment by ElenaFdR** [comment]
Folks, I plan to close this branch. Rather than revert #99, I'll just fix the errors I introduced in it. That seems like it will be faster than unpicking this revert PR (especially given all the changes made to develop in the meantime).

**Comment by angelplasma** [comment]
> @angelplasma My sense is that "#_" would be better yet that also seems too broad as we now have the hidden trap of not checking a link like "example.com/internal_functions#_foo". I believe to match exactly "#" the regular expression would require anchors on either side, e.g.: "/^#$/" as described in https://ruby-doc.org/core-2.7.1/Regexp.html
> 
> Is there a branch pushed which has the types of links which are failing which we can test against?

I hear that. Thank you for this insight + info!

Since (per Elena) we're leaving this branch behind, and we want to follow through on a proper solve for the "#", I created a specific issue for it here: https://github.com/publiccodenet/publiccode.net/issues/140

As Elena mentioned earlier, the relevant (now merged, but not yet live) example is here: https://github.com/publiccodenet/publiccode.net/pull/111

**Comment by ElenaFdR** [comment]
Superseded by #141.
